### ISSUE TITLE
How to download the attachments?

### ISSUE THREAD

**Issue by Karie0910** [dgunning/edgartools#61]
I am trying to download the attachments.
`
filings = get_filings(range(2022, 2025), form="NPORT-P", amendments=True)
filings[0].attachments.download(path=r'D:/Karie')
`
Then there are some errors:
![image](https://github.com/dgunning/edgartools/assets/172482849/11f821d4-8b90-4efb-a2c8-35da8cd12125)
How to solve this? 
Thank you!

**Comment by dgunning** [comment]
Hello Karie,

Can you select the specific filing you are trying to download e.g.

```python
str(filings[0])
"Filing(form='NPORT-P', filing_date='2024-06-11', company='AETOS DISTRESSED INVESTMENT STRATEGIES FUND LLC', cik=1169578, accession_no='0001104659-24-070456')"

```

Although since a day has passed filings[0] will likely have changed. In any even see if you can get a single identified filing to fail

**Comment by dgunning** [comment]
The original issue with the error downloading the attachments could possibly be an encoding bug.

I'm making a change to set the encoding explicitly as 'utf-8' and will release a new version in a day or two

**Comment by dgunning** [comment]
Released version 2.24.2 which explicitly sets the encoding to 'utf-8'. Please test

**Comment by Karie0910** [comment]
Thank you! It is working now.
